README.md

```diff
@@ -41,24 +40,25 @@ predicament of when to actually upload the image to ECR
-| task_policies | List of IAM Policy ARNs to attach to the task execution IAM Policy| [] |
-| min_capacity | Minimum task count for autoscaling | 1 |
-| max_capacity | Maximum task count for autoscaling | 2 | 
+| Name | Type | Description | Default |
+| --- | --- | --- | --- |
+| app_name | string | Application name to name your Fargate API and other resources | |
+| env | string | Environment of the AWS Account (e.g. dev, prd) | |
+| dockerfile_dir | string | The directory that contains the Dockerfile to be built and then uploaded to ECR | |
+| image_port | number | The port the docker image is listening on | |
+| health_check_path | string | Health check path for the image | "/" |
+| container_env_variables | map(string) | Map of environment variables to pass to the container definition | {} |
+| container_secrets | map(string) | Map of secrets from the parameter store to be assigned to env variables. Use `task_policies` to make sure the Task's IAM role has access to the SSM parameters | {} |
+| task_policies | list(string) | List of IAM Policy ARNs to attach to the task execution IAM Policy| [] |
+| min_capacity | number | Minimum task count for autoscaling | 1 |
+| max_capacity | number | Maximum task count for autoscaling | 2 | 
 
 ## Outputs
-| Name | Description |
-| --- | --- |
-| fargate_service | Fargate ECS Service [object](https://www.terraform.io/docs/providers/aws/r/ecs_service.html#attributes-reference) |
-| fargate_service_security_group | Security Group [object](https://www.terraform.io/docs/providers/aws/r/security_group.html#attributes-reference) assigned to the Fargate service |
-| codedeploy_appspec_json | JSON string of a simple appspec.json file to be used in the CodeDeploy deployment |
-| alb | The Application Load Balancer (ALB) [object](https://www.terraform.io/docs/providers/aws/r/lb.html#attributes-reference) |
-| alb_security_group | The ALB's security group [object](https://www.terraform.io/docs/providers/aws/r/security_group.html#attributes-reference) |
-| dns_record | The DNS A-record mapped to the ALB | 
+| Name | Type | Description |
+| --- | --- | --- |
+| fargate_service | object | Fargate ECS Service [object](https://www.terraform.io/docs/providers/aws/r/ecs_service.html#attributes-reference) |
+| fargate_service_security_group | object | Security Group [object](https://www.terraform.io/docs/providers/aws/r/security_group.html#attributes-reference) assigned to the Fargate service |
+| codedeploy_appspec_json | string | JSON string of a simple appspec.json file to be used in the CodeDeploy deployment |
+| alb | object | The Application Load Balancer (ALB) [object](https://www.terraform.io/docs/providers/aws/r/lb.html#attributes-reference) |
+| alb_security_group | object | The ALB's security group [object](https://www.terraform.io/docs/providers/aws/r/security_group.html#attributes-reference) |
+| dns_record | object | The DNS A-record mapped to the ALB | 
```

Comment:
Pull the object links over into the type column.

Reply:
Done